apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -105,12 +105,28 @@ type VSHNDBaaSSizeSpec struct {
 	// Memory defines the amount of memory in units of bytes for an instance.
 	Memory string `json:"memory,omitempty"`
 
+	// Requests defines CPU and memory requests for an instance
+	Requests VSHNDBaaSSizeRequestsSpec `json:"requests,omitempty"`
+
 	// +kubebuilder:default="5Gi"
 
 	// Disk defines the amount of disk space for an instance.
 	Disk string `json:"disk,omitempty"`
 }
 
+// VSHNDBaaSSizeRequestsSpec contains settings to control the resoure requests of a service.
+type VSHNDBaaSSizeRequestsSpec struct {
+	// +kubebuilder:default="600m"
+
+	// CPU defines the amount of Kubernetes CPUs for an instance.
+	CPU string `json:"cpu,omitempty"`
+
+	// +kubebuilder:default="3500Mi"
+
+	// Memory defines the amount of memory in units of bytes for an instance.
+	Memory string `json:"memory,omitempty"`
```

Comment:
Do we actually want to set defaults for these. If I understand the `SGInstanceProfile` correctly setting to request basically means that they're the same as the limit. Which is IMO more intuitive than using a default.

Reply:
It's actually the other way around. Setting the limits and leaving the requests empty will set the requests to the same value as the limits. But I agree with you. I removed the defaults.